frontend/src/views/RecruitmentRegister.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="recruitment-register">
+    <Form @submit.prevent="submit">
+      <h1>지원서 작성</h1>
+
+      <TextField
+        v-for="(item, index) in recruitmentItems" v-bind:key="item.id"
+        v-model="recruitmentItemInputs[index]"
+        name="recruitment-item"
+        type="textarea"
+        :label="item.title"
+        :description="item.description"
+        placeholder="내용을 입력해주세요."
+        :rules="rules.recruitmentItem"
+        :maxLength="item.maximumLength"
```

Comment:
```suggestion
        :max-length="item.maximumLength"
```

모든 속성은 kebab-case를 사용해주세요.